qa/L0_response_cache/test.sh

```diff
@@ -31,11 +31,42 @@ TEST_LOG="./response_cache_test.log"
 UNIT_TEST="./response_cache_test --gtest_output=xml:response_cache.report.xml"
 export CUDA_VISIBLE_DEVICES=0
 
+REPO_VERSION=${NVIDIA_TRITON_SERVER_VERSION}
+if [ "$#" -ge 1 ]; then
+    REPO_VERSION=$1
+fi
+if [ -z "${REPO_VERSION}" ]; then
+    echo -e "Repository version must be specified"
+    echo -e "\n***\n*** Test Failed\n***"
+    exit 1
+fi
+if [ ! -z "$TEST_REPO_ARCH" ]; then
+    REPO_VERSION=${REPO_VERSION}_${TEST_REPO_ARCH}
+fi
 # Only localhost supported in this test for now, but in future could make
 # use of a persistent remote redis server, or similarly use --replicaof arg.
 export TRITON_REDIS_HOST="localhost"
 export TRITON_REDIS_PORT="6379"
 REDIS_LOG="./redis-server.unit_tests.log"
+ENSEMBLE_CACHE_TEST_PY="./ensemble_cache_test.py"
+SERVER=/opt/tritonserver/bin/tritonserver
+# SERVER_ARGS="--model-repository=`pwd`/models --response-cache-byte-size=8192"
```

Comment:
Do we need this line?

Reply:
we don't need. probably missed in removing it.